### DISCUSSION TITLE
Internal logging grows and consumes space

### DISCUSSION THREAD

**Discussion by stevenmunro** [quirrel-dev/quirrel#1010] [General]
I have to manually stop and restart my "quirrel/redis" docker-compose because something is chewing up disk space, and I assume it must be logging inside the quirrel docker container. Once I restart the container, I get my space back.

Is this a known issue?

**Comment by Skn0tt** [comment]
Not a known issue, no. Quirrel is not logging to a file, but to stdout - if it really is logging (Quirrel is logging quite a lot, that's true), maybe you can select a different logging provider with Docker?
